perses/utils/openeye.py

```diff
@@ -420,7 +420,10 @@ def generate_unique_atom_names(molecule):
     else:
         # generating new atom names
         from collections import defaultdict
-        from simtk.openmm.app.element import Element
+        try:
+            from openmm.app.element import Element
+        except ModuleNotFoundError:  # <=7.5.0
+            from simtk.openmm.app.element import Element
```

Comment:
According to @peastman this version should work for both?

```suggestion
            from simtk.openmm.app import Element
```

Reply:
But I don't know if we should leave the try/except version anyway in case we want to provide support for all OpenMM  namespaces without pinned version constraints...

Reply:
I don't know enough about the `perses` user base to know if that would be an issue or not. I know that sometimes depending on the packages used in the project, it could be impossible to make all the packages happy. 

@jaimergp @jchodera 

Are we okay to merge this PR even though the nightly is failing? I'm looking into seeing if the changes planned for openmm can be made more backwards compatible: 
https://github.com/openmm/openmm/issues/3022

But in order to for nightly to pass, we need to make all the packages that depend on `perses` also work with nightly and that may take a while. I really like testing against nightly, but I think for now we can make the nightly tests as "okay to merge if everything else passes"

Reply:
I'd be ok with that, specially since the remaining failures are not part of this codebase, and we can always open a new PR.

Reply:
Actually, there's a lot of `import simtk` statements in the code that will need to be fixed or try/except guarded at some point. You can do it in this PR or maybe in a separate one for better traceability.

Reply:
To be more clear, I'd merge now and deal with the OpenMM namespace issues separately, since 7.6 won't happen for some months and we have more pressing issues, I'd say. @jchodera, do you agree?

Reply:
:+1: that is what I was thinking too, this PR will become too big if we wait for things to be **perfect**

Reply:
I'm going to merge this in, we can undo it if it is an issue